scripts/test_analyzer/pytest_marker_analyzer.py

```
@@ -0,0 +1,2635 @@
+
+        for test_name in tests:
+            try:
+                rel_path = test_file.relative_to(repo_root)
+                node_id = f"{rel_path}::{test_name}"
+                results.append((node_id, test_name, test_file))
+            except ValueError:
+                pass
+
+    except Exception:
+        # Silently skip files with errors
+        pass
```

Ruff (0.14.10)

`try`-`except`-`pass` detected, consider logging the exception

(S110)